telegram/_chatpermissions.py

```diff
@@ -121,3 +121,14 @@ def __init__(
             self.can_invite_users,
             self.can_pin_messages,
         )
+
+    @classmethod
+    def all_true(cls) -> 'ChatPermissions':
```

Comment:
so do we need a `all_false` method too? Also I did prefer the clarity of `unrestrict` better. More suggestions:

`allow_all`/`disallow_all`
`all_permissions`/`no_permissions`

Reply:
> so do we need a all_false method too?

That is currently the assumed default by telegram, but sure, we can add that.

> Also I did prefer the clarity of unrestrict better

I don't like it because it is implicit design. If Telegram ever changes the behaviour (doubtfully, still), we would have a legacy method name dragging around which will make no sense if used in a different project.

> more suggestions

If you'd really like to rename the method, I like `all_permissions`/`no_permissions` best.

Reply:
>  I like all_permissions/no_permissions best.

let's go for this then